src/CommunityToolkit.Maui.Core/Platform/KeyboardManager/KeyboardManager.ios.cs

```diff
@@ -0,0 +1,13 @@
+using System;
+using UIKit;
+
+namespace CommunityToolkit.Maui.Core.Platform;
+
+public static partial class KeyboardManager
```

Comment:
Please check this file is included. We combine iOS and macOS with *.macios.cs files